### PR TITLE
Status bar: fix dropdown width

### PR DESCRIPTION
**Pull request by pedropintosilva**
They have now a flex box so we should allow them to grow but not to
shrink.

Signed-off-by: Pedro Pinto Silva <pedro.silva@collabora.com>
Change-Id: Ide87f81d403e1f77977c6098d6f1a6e85828468e

### CODE REVIEW

**Comment by pedropintosilva** [comment]
Probably the cypress helper `makeZoomItemsVisible` needs update

```javascript
function makeZoomItemsVisible() {
	cy.log('>> makeZoomItemsVisible - start');

	cy.cGet('#toolbar-down #zoomin')
		.then(function(zoomInItem) {
			if (!Cypress.dom.isVisible(zoomInItem)) {
				cy.cGet('#toolbar-down .ui-scroll-right').click();
			}
		});

	cy.cGet('#toolbar-down #zoomin').should('be.visible');

	cy.log('<< makeZoomItemsVisible - end');
}
```

**Comment by pedropintosilva** [comment]
nop rebasing this and re-runing make -C cypress_test run-desktop spec=calc/statusbar_spec.js I got everything green

**Comment by pedropintosilva** [comment]
it fails in CI on 

Tests:        7                                                                                │
  │ Passing:      6                                                                                │
  │ Failing:      1                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  2                                                                                │
  │ Video:        false                                                                            │
  │ Duration:     36 seconds                                                                       │
  │ Spec Ran:     writer/searchbar_spec.js       


the same locally:

image

:shrug:

**Comment by eszkadev** [comment]
it was fixed this week on CI I think - please rebase

**Review by eszkadev — APPROVED** [review]

**Comment by pedropintosilva** [comment]
now it failed in another one, I have retry again

File: cypress_test/integration_tests/common/desktop_helper.js [hunk 1]
```diff
@@ -181,13 +181,20 @@ function shouldHaveZoomLevel(zoomLevel) {
...
-			if (!Cypress.dom.isVisible(zoomInItem)) {
-				cy.cGet('#toolbar-down .ui-scroll-right').click();
+	const scrollRight = () => {
+		cy.cGet('#toolbar-down .ui-scroll-right').then($scrollRightButton => {
+			if ($scrollRightButton.is(':visible')) {
+				// Wrap .ui-scroll-right so we can continue executing commands
+				cy.wrap($scrollRightButton).click();
+				// Wait the same ms as in Util.ScrollableBar.ts timeout
+				cy.wait(350);
+				// Scroll again if $scrollRightButton is visible
```
[the remark this answers is not in the document]
Comment (pedropintosilva) [reply cypress_test/integration_tests/common/desktop_helper.js:191]: no, scroll again if the right scroll indicator is visible, meaning: keep scrolling (clicking in it) if there is a $scrollRightButton in the screen

**Pull request merged**